solution/solver.go

```diff
@@ -30,6 +30,7 @@ type PlanningRequest struct {
 	Location     POI.Location
 	Slots        []SlotRequest
 	Weekday      POI.Weekday
+	TravelDate   string
```

Comment:
@weihesdlegend To help saving plan with the **correct** travel date on the show page, I added `TravelDate` in `PlanningRequest` struct.